dds_cli/project_info.py

```diff
@@ -44,37 +44,13 @@ def __init__(
         self.project = project
 
     # Public methods ###################### Public methods #
-    def get_project_info(self):
-        """Collect project information from API."""
-        # Get info about a project from API
-        response, _ = dds_cli.utils.perform_request(
-            DDSEndpoint.PROJ_INFO,
-            method="get",
-            headers=self.token,
-            params={"project": self.project},
-            error_message="Failed to get project information",
-        )
-
-        project_info = response.get("project_info")
-        if not project_info:
-            raise dds_cli.exceptions.ApiResponseError(message="No project information to display.")
```

Comment:
Atm if "project_info" is not returned for some reason, a traceback is printed. I think we should add this again to the `get_project_info` method in `base.py`. In order to let the users delete or archive the project even if the project info is not displayed, we can add a `try-except` in `update_status` around line 113, and the `except` can print out for example "Could not display project information at this time." and then move on to the confirmation, exactly like you have it now. 

Thoughts / other suggestions on that?

Reply:
**Additional comment that has nothing to do with this PR - don't do this in this PR, it's outside of the scope:**
Future task: Catch the exception raised in the `get_project_info` method where it's raised, if we don't want it to completely quit.
- `show_project_info`, line 52: Should **not** catch the exception, it should still be raised
- `update_info`, line 70: Should catch the exception with a try-except. The except could for example print and ask for confirmation from the user: "Could not display project information. Are you sure you want to change the project info?"

Reply:
I removed it mainly because the requests was already throwing an exception, so, for example, when the request fails it already leaves the function. The coverage was never getting to 100% because of that as well. The _if not project_info_: was never being executed. I can add another try-catch in this method if we want to generalise every problem in this requests through the _ApiResponseError_

Reply:
Yeah I see what you mean, this check is more to verify that "project_info" actually is returned, so that if the API returns something, but not the term "project_info", then it prints out a message that we can easily interpret instead of a traceback that might also result in support tickets etc. So it's good to keep this check anyway